specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/appplatform.json

```diff
@@ -3408,6 +3408,10 @@
           "description": "Indicate if only https is allowed.",
           "type": "boolean"
         },
+        "enableEndToEndTLS": {
+          "description": "Indicate if end to end tls is allowed.",
```

Comment:
`end to end tls is allowed` -> `end to end tls is enabled` ?

Reply:
-> `end-to-end TLS` ?

Reply:
thanks
new wording: "Indicate if end to end TLS is enabled."